### DISCUSSION TITLE
api request limit

### DISCUSSION THREAD

**Discussion by apujayed** [pocketbase/pocketbase#5806] [Q&A]
can we restrict the limit maximum of 10 requests per second per user?

**Comment by dschissler** [comment]
It seems quite advanced.  This is from 0.23 in app settings.  0.22 probably has this as well.

![Screenshot from 2024-11-05 23-39-14](https://github.com/user-attachments/assets/0dc2e3b0-32df-4654-b921-b30803cc384d)

**Reply by dschissler** [reply]
There is even a pop-up tutorial on it.

![Screenshot from 2024-11-05 23-43-31](https://github.com/user-attachments/assets/b6ee13f6-6de3-41ac-b533-9338b0c90799)

**Comment by apujayed** [comment]
okay thanks

**Comment by apujayed** [comment]
![Screenshot (623)](https://github.com/user-attachments/assets/6ebafbea-b919-4fff-af9b-c7f1ba2486eb)
i updated the version 0.22.23.but i dont found this option you mentioned

**Reply by ganigeorgiev** [reply]
The builtin rate limiter is available only with PocketBase v0.23.0-rc.

For <= v0.22.x you can specify rate limiting rules on reverse proxy (nginx, apache, caddy, etc.) level.
